easypermissions/src/main/java/pub/devrel/easypermissions/EasyPermissions.java

```diff
@@ -352,7 +352,7 @@ private static Activity getActivity(@NonNull Object object) {
         } else if (object instanceof android.app.Fragment) {
             return ((android.app.Fragment) object).getActivity();
         } else {
-            return null;
+            throw new IllegalArgumentException("Context cannot be null.");
```

Comment:
I changed it to be `IllegalArgumentException` since it is now called immediately using the method's argument instead of further down the stack.